Codeinwp/neve
# [🎛 Customizer UI][Feature] Visual UI/UX Refactor

### Summary
- chore: Removed some unused code (the Top_Bar class)
- chore: change all toggle controls to react based controls
- fix: about page links were linking to non-existent customizer sections

- chore: UI refactor for the sizing control
- chore: UI refactor for the single sizing component
- chore: update react range control and responsive range components
- chore: replace all range settings with React-based ones
- chore: adds line-height units to typography controls
- chore: refactors the typography controls appearance
- chore: refactors the button appearance controls


### Will affect visual aspect of the product

NO

### Test instructions

- Should migrate properly from older versions
- Should work with [this PR](https://github.com/Codeinwp/neve-pro-addon/pull/585)

- Check that all theme defined checkboxes / old toggles are now react-based toggle controls.
- Check that all sizing controls work as they were supposed.
- Test that all ranges inside the customizer work properly and as before
- Test that all typography customizer options migrate properly and work as expected
- Test that the button appearance controls migrate properly


Closes #1394, #1395, #1396, #1397, #1398.
